src/components/search/SearchItem.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import style from './SearchItem.module.css';
+import { useNavigate } from 'react-router-dom';
+import { useState } from 'react';
+
+const SearchItem = ({ id, img, title, price, description }) => {
+  const navigate = useNavigate();
+  const [heart, setHeart] = useState(false);
+  function onClick() {
+    setHeart((cur) => !cur);
+  }
+
+  return (
+    <li
+      className={style.item}
+      onClick={(e) => {
+        if (e.target.nodeName !== 'BUTTON') {
+          navigate(`/products/${id}`, { state: { id, title, img, price } });
+        }
+      }}
+    >
+      <div className={style.container}>
+        <img className={style.img} src={img} alt={title} />
+        <div>
+          <p className={style.title}>{title}</p>
+          <p className={style.price}>{price.toLocaleString()}원</p>
+          <p className={style.description}>{description}...</p>
+        </div>
+      </div>
+
+      <div>
+        <button onClick={onClick} className={heart ? style.btn_heart__red : style.btn_heart}>
```

Comment:
```suggestion
        <button onClick={onClick} className={heart ? style.btnHeart_red : style.btnHeart}>
```
[컨벤션](https://www.notion.so/hnhyerim/code-updated-12-22-f4f8aa0f6de9424ab2cad142df625d83#90802f6515ae4e219554b0c881b0aca2)